### DISCUSSION TITLE
Avalonia ui中使用CommunityToolkit.Mvvm报错

### DISCUSSION THREAD

**Discussion by xzf888** [BYJRK/DotNet-Discussions#185] [MVVM]
axaml:
**\<ComboBox Items="{Binding ListItemsCollection}" SelectedIndex="0" />**

C#
[ObservableProperty]
public ObservableCollection<string> listItemsCollection = new() ;

报错信息:
AVLN:0004	Unable to find suitable setter or adder for property Items of type Avalonia.Controls:Avalonia.Controls.ItemsControl for argument Avalonia.Markup.Xaml:Avalonia.Markup.Xaml.MarkupExtensions.CompiledBindingExtension, available setter parameter lists are:System.Object

**Comment by BYJRK** [comment]
绑定 `ItemsSource` 试试呢？

**Comment by xzf888** [comment]
感谢,试了下可以了
